src/nsntrace.c

```diff
@@ -177,8 +177,8 @@ _nsntrace_start_tracee(struct nsntrace_options *options)
 		uid = getuid();
 		gid = getgid();
 	}
-	setgid(gid);
-	setuid(uid);
+	uid = setgid(gid);
+	gid = setuid(uid);
```

Comment:
Thanks!

But this not really make sense. These functions set the group id and user id of the calling process.
They return 0 on success and -1 on failure. So to handle this warning we would need to do something like:

if (setgid() < 0) { ... }

And maybe the commit message can say what warning we are fixing?